vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{
+            "VPD Command line tool to dump the inventory, read and "
+            "update the keywords,\nto fix mismatch keyword's and reset keyword's value to its default.\n"};
+
+        std::string l_objectPath{};
+        std::string l_recordName{};
+        std::string l_keyword{};
+        std::string l_filePath{};
+
+        auto l_objectOption = l_app.add_option("--object, -O", l_objectPath,
+                                               "Enter the Object Path");
+        auto l_recordOption = l_app.add_option("--record, -R", l_recordName,
+                                               "Enter the Record Name");
```

Comment:
Can we have the description something like:
"Use this option to provide VPD record's name".

Reply:
Description is updated, keeping shorter description for all options.